.github/workflows/snap.yml

```diff
@@ -16,8 +16,8 @@ jobs:
       matrix:
         platform:
         - amd64
-        - armhf
-        - arm64
+#        - armhf
+#        - arm64
```

Comment:
You can get arch builds with:

https://github.com/canonical/actions/blob/daa9ae6d31b2a90eaac93215ec357b0cee66f081/build-snap/action.yml#L12-L27

Reply:
Ack. I don't immediately know how to use that (nor why it was not needed before). Will have a look later

Reply:
With `@multiarch` it built locally (with docker-through-qemu), but that's slow and unreliable. The new way builds with `snapcraft remote-build`, but requires Launchpad credentials for this. I've now added mir-ci-bot's creds so should be good.

Reply:
Thanks! (The bit I was missing was generating credentials with the right stuff.)